src/dx/versioning.h

```diff
@@ -26,14 +45,14 @@ typedef struct VanillaPlayerData {
     /* 0x008 */ s8 maxBP;
     /* 0x009 */ s8 level;
     /* 0x00A */ b8 hasActionCommands;
-    /* 0x00B */ char pad_00B;
+    /* 0x00B */ unsigned char pad_00B;
```

Comment:
Why?

Reply:
Other unused/padding fields in structs are usually typed unsigned char, with signed char (arrays) only used for strings. This change is for consistency.